### ISSUE TITLE
[core]Line number rendering problem

### ISSUE THREAD

**Issue by yu6315** [skylot/jadx#1153]
When changing the line number display style or scrolling code, the line number display has problems, the background color and UI display are confused, and there is no problem in SmalI mode.like this. #404 ,I use the latest version of exe will still have this problem.
![line-number](https://user-images.githubusercontent.com/48148334/114998061-24588080-9ed3-11eb-8c18-aec5cbd32c64.gif)

Thanks.

**Comment by yu6315** [comment]
![line-number](https://user-images.githubusercontent.com/48148334/115001300-1bb57980-9ed6-11eb-80bd-d9241e67e677.gif)

**Comment by jpstotz** [comment]
The code style looks like Jadx "dark mode" as editor theme. What OS and Java runtime version + vendor do you use?

**Comment by yu6315** [comment]
> The code style looks like Jadx "dark mode" as editor theme. What OS and Java runtime version + vendor do you use?


Yes, I used "dark mode", And I still have problems using the "default mode", OS is Windows 10 HOME 20H2, Java version is java 11.0.10 2021-01-19 LTS, Oracle Corporation.
![image](https://user-images.githubusercontent.com/48148334/115009838-4ce67780-9edf-11eb-807c-81c19fcb5155.png)

**Comment by jpstotz** [comment]
Interesting, I have the identical environment with the only difference that I am using AdoptOpenJDK 11 instead of Oracle and I can not reproduce your issue using latest unstable and stable Jadx version.

**Comment by FantasqueX** [comment]
I met an even worse problem with line number rendering
![image](https://user-images.githubusercontent.com/46747123/120889511-18b24c00-c630-11eb-9b99-d903564d8565.png)

I'm using Microsoft build openJDK 16 on Windows 10 19043

```
openjdk 16.0.1 2021-04-20
OpenJDK Runtime Environment Microsoft-22300 (build 16.0.1+9)
OpenJDK 64-Bit Server VM Microsoft-22300 (build 16.0.1+9, mixed mode, sharing)
```

**Comment by skylot** [comment]
@FantasqueX can you check [latest unstable build](https://nightly.link/skylot/jadx/workflows/build/master)? I made a few fixes in 4732fa36a602d064ac3f543f2fd09aee43450c25.

**Comment by FantasqueX** [comment]
> @FantasqueX can you check [latest unstable build](https://nightly.link/skylot/jadx/workflows/build/master)? I made a few fixes in [4732fa3](https://github.com/skylot/jadx/commit/4732fa36a602d064ac3f543f2fd09aee43450c25).

I checked the latest build just now. The problem has been fixed on my PC. Thank you for your work!

**Comment by skylot** [comment]
> The problem has been fixed on my PC

Great! Thanks for confirm!

I will close issue as fixed.
